src/batch-client/index.ts

```diff
@@ -590,7 +590,7 @@ export class ZimbraBatchClient {
 	public createContact = (data: CreateContactInput) =>
 		this.jsonRequest({
 			name: 'CreateContact',
-			body: createContactBody(data),
+			body: createContactBody(data, this.localStoreClient),
```

Comment:
we should pass boolean from here, instead of object

Reply:
Fixed.